src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -145,69 +152,72 @@
                                 <DataTemplate>
                                     <Grid>
                                         <vsImaging:CrispImage Height="16" Width="16"
-                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
-                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
+                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}"
+                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}" />
                                         <vsImaging:CrispImage Height="16" Width="16"
                                                       Margin="-5,0,0,0"
                                                       ToolTipService.ToolTip="Location is navigable. Use double-click or Enter to open the document."
-                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentSource}" 
-                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToVisibilityConverter}, Mode=OneWay}"/>
+                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentSource}"
+                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToVisibilityConverter}, Mode=OneWay}" />
                                     </Grid>
                                 </DataTemplate>
                             </DataGridTemplateColumn.CellTemplate>
                         </DataGridTemplateColumn>
                         <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.RuleKey" Header="Code" Width="130" MinWidth="54">
                             <DataGridTemplateColumn.CellTemplate>
                                 <DataTemplate>
-                                    <TextBlock Text="{Binding TaintIssueViz.Issue.RuleKey}" 
-                                       Style="{StaticResource CellTextBlockStyle}" />
+                                    <TextBlock>
+                                        <Hyperlink Style="{StaticResource RuleKeyLinkStyle}" CommandParameter="{Binding TaintIssueViz.Issue.RuleKey}" Command="{Binding DataContext.NavigateToRuleDescriptionCommand, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=DataGrid}}">
```

Comment:
wasn't it supposed to reference vs' styles?

Reply:
I have created another ticket for that so we do not waste time with theming.